### ISSUE TITLE
btjd_to_astropy_time() does not accept some float arrays

### ISSUE THREAD

**Issue by orionlee** [KeplerGO/lightkurve#607]
#### Problem description

Per [documentation](https://docs.lightkurve.org/api/lightkurve.utils.btjd_to_astropy_time.html#lightkurve.utils.btjd_to_astropy_time), `utils.btjd_to_astropy_time()` is supposed to accept an array of floats.

However, the function only accepts some types of arrays. See example below.

So either the documentation is inaccurate, or the implementation has bugs.

I expect the resolution would also apply to the peer `bkjd_to_astropy_time()`

#### Example

```python
import numpy as np
from lightkurve.utils import btjd_to_astropy_time

t1 = [1.1, 2.2, 3.3]
btjd_to_astropy_time(t1) # failed 

t2 = np.array(t1, dtype='float')
btjd_to_astropy_time(t2) # worked
```
The error is:
```
--> 370     jd = btjd + bjdref
...
TypeError: can only concatenate list (not "float") to list
```

#### Environment

-  platform (e.g. Linux, OSX, Windows): Windows, Jupyter, Python 3.7
-  lightkurve version (e.g. 1.0b6): 1.2.0
-  installation method (e.g. pip, conda, source): conda

**Comment by barentsen** [comment]
Thanks @orionlee!  Once #607 is merged, these functions will accept lists/tuples/floats in addition to a numpy array!

**Comment by barentsen** [comment]
Fixed by #609!
